server/upgrade_risks_prediction.go

```diff
@@ -0,0 +1,131 @@
+// Copyright 2023 Red Hat, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package server
+
+import (
+	"encoding/json"
+	"io"
+	"net/http"
+
+	httputils "github.com/RedHatInsights/insights-operator-utils/http"
+	"github.com/RedHatInsights/insights-operator-utils/responses"
+	utypes "github.com/RedHatInsights/insights-operator-utils/types"
+	"github.com/RedHatInsights/insights-results-smart-proxy/types"
+
+	"github.com/rs/zerolog/log"
+)
+
+// UpgradeRisksPredictionServiceEndpoint endport for the upgrade prediction service
+const UpgradeRisksPredictionServiceEndpoint = "upgrade-risks-prediction/cluster/{cluster}"
```

Comment:
same discussion we had a few days ago...
should be `clusters/{cluster}/upgrade-risks-prediction` to be compliant with the rest of the api

Reply:
I believe this is the endpoint of the prediction service. The smart-proxy endpoints seems ok https://github.com/RedHatInsights/insights-results-smart-proxy/pull/1007/files#diff-0a5cb9b87202b05d6f48145746f515c38ec3f96af25522563319cf4c00d8ca28R36

Reply:
I think I misunderstood this... sorry!

Reply:
so does somebody about to fix that one in the other service?

Reply:
Do you mean to put the `cluster/{cluster}` before the `upgrade-risks-prediction`, right?